pyproject.toml

```diff
@@ -8,7 +8,7 @@ readme = "README.md"
 packages = [{include = "python_hooks"}]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "~3.9"
```

Comment:
doesn't this break anything that uses python >3.9?

Reply:
Ah right, thanks.